transactions/src/main/java/org/jboss/as/txn/logging/TransactionLogger.java

```diff
@@ -250,4 +250,7 @@
     @Message(id = 38, value = "Unexpected error on suspending transaction for work %s")
     RuntimeException cannotSuspendInflowTransactionUnexpectedError(Work txn, @Cause Exception e);
 
+    @LogMessage(level = WARN)
+    @Message(id = 39, value = "The transaction timeout has been set to %s while the value is 0")
```

Comment:
I'm not really user I understand this message. Specifically "while the value is 0".

Reply:
Nor do I, @zhfeng can you take a look please?

Reply:
@jamezp @tomjenkinson This message is warning the user that the transaction timeout can not be set to "0" and will replace with the maximum-timeout attribute.

Reply:
@zhfeng Those attributes should probably be listed then. Just so it's clear for the user.